test_unstructured/partition/test_json.py

```diff
@@ -0,0 +1,98 @@
+import os
+import pathlib
+
+import pytest
+
+from unstructured.partition.auto import partition
+from unstructured.partition.json import partition_json
+from unstructured.staging.base import elements_to_json
+
+DIRECTORY = pathlib.Path(__file__).parent.resolve()
+
+test_files = [
+    "fake-text.txt",
+    "layout-parser-paper-fast.pdf",
+    "fake-html.html",
+    "fake.doc",
+    "fake-email.eml",
+    "fake-power-point.ppt",
+    "fake.docx",
+    "fake-power-point.pptx",
+]
+
+
+@pytest.mark.parametrize("filename", test_files)
+def test_partition_text_from_filename(filename: str):
+    # root, _ = os.path.splitext(filename)
+
+    path = os.path.join(DIRECTORY, "..", "..", "example-docs", filename)
+    elements = partition(filename=path)
+
+    test_path = os.path.join(DIRECTORY, "test_json_output", filename + ".json")
```

Comment:
you can place these .json's in a temporary directory that automatically gets cleaned up by using something like:
https://github.com/Unstructured-IO/unstructured/blob/601f250/unstructured/partition/doc.py#L40

Reply:
related: it looks like all the empty .json files may be removed from this PR